src/main/resources/db/migration_h2/V4__foreign_keys.sql

```diff
@@ -0,0 +1,250 @@
+-- account
+
+-- h2 required not composite key for link foreign key
+create index ACCOUNT_ID_INDEX
+	on "account" ("id");
+
+-- account_asset
+
+alter table "account_asset"
+	add constraint ACCOUNT_ASSET_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+alter table "account_asset"
+	add constraint ACCOUNT_ASSET_ASSET_ID_FK
+		foreign key ("asset_id") references "asset" ("id")
+			on delete cascade;
+
+-- alias
+
+alter table "alias"
+	add constraint ALIAS_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+-- alias_offer
+
+alter table "alias_offer"
+	add constraint ALIAS_OFFER_ACCOUNT_ID_FK
+		foreign key ("buyer_id") references "account" ("id")
+			on delete cascade;
+
+-- ask_order
+
+alter table "ask_order"
+	add constraint ASK_ORDER_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+alter table "ask_order"
+	add constraint ASK_ORDER_ASSET_ID_FK
+		foreign key ("asset_id") references "asset" ("id")
+			on delete cascade;
+
+alter table ask_order
+	add constraint ask_order_asset_id_fk
+		foreign key (asset_id) references asset (id)
+			on delete cascade;
+
+-- asset
+
+alter table "asset"
+	add constraint ASSET_ASSET_ID_FK
+		foreign key ("account_id") references "asset" ("id")
+			on delete cascade;
+
+-- asset_transfer
+
+alter table "asset_transfer"
+	add constraint ASSET_TRANSFER_ACCOUNT_ID_FK
+		foreign key ("sender_id") references "account" ("id")
+			on delete cascade;
+
+alter table "asset_transfer"
+	add constraint ASSET_TRANSFER_ACCOUNT_ID_FK_2
+		foreign key ("recipient_id") references "account" ("id")
+			on delete cascade;
+
+alter table "asset_transfer"
+	add constraint ASSET_TRANSFER_ASSET_ID_FK
+		foreign key ("asset_id") references "asset" ("id")
+			on delete cascade;
+
+-- at
+
+alter table "at"
+	add constraint AT_ACCOUNT_ID_FK
+		foreign key ("creator_id") references "account" ("id")
+			on delete cascade;
+
+-- at_state
+
+alter table "at_state"
+	add constraint AT_STATE_AT_ID_FK
+		foreign key ("at_id") references "at" ("id")
+			on delete cascade;
+
+-- bid_order
+
+alter table "bid_order"
+	add constraint BID_ORDER_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+alter table "bid_order"
+	add constraint BID_ORDER_ASSET_ID_FK
+		foreign key ("asset_id") references "asset" ("id")
+			on delete cascade;
+
+-- block
+-- no generator_id foreign key because account may be not yet created
```

Comment:
or account with generator_id always created before creating block?

Reply:
I can change so that block is inserted after account table is updated.

Reply:
bb98740d19c7fff9d06a0347048fd1adb4dad256

Reply:
I'm not sure good idea insert block after accounts/transactions and etc. Block is parent, all other is his childs. https://github.com/burst-apps-team/burstcoin/pull/369#issuecomment-599256284